src/locales/ko/mystery-encounters/global-trade-system-dialogue.json

```
@@ -1,32 +1,32 @@
 {
-  "intro": "It's an interface for the Global Trade System!",
-  "title": "The GTS",
-  "description": "Ah, the GTS! A technological wonder, you can connect with anyone else around the globe to trade Pokémon with them! Will fortune smile upon your trade today?",
-  "query": "What will you do?",
+  "intro": "GTS 단말기가 있다!",
+  "title": "GTS",
+  "description": "오, GTS! 세상 참 좋아졌네요.\n이렇게 전세계 사람들과 연결해서 포켓몬을 교환 할 수 있으니 말이에요!\n오늘은 어떤 흥미로운 교환이 이루어질까요?",
+  "query": "어떻게 하시겠습니까?",
   "option": {
     "1": {
-      "label": "Check Trade Offers",
-      "tooltip": "(+) Select a trade offer for one of your Pokémon",
-      "trade_options_prompt": "Select a Pokémon to receive through trade."
+      "label": "교환 제안을 확인한다",
+      "tooltip": "(+) 포켓몬 중 하나와 제안을 골라 교환",
```

```suggestion
      "tooltip": "(+) 포켓몬 1마리를 제안을 골라 교환",
```
이게 좀 더 명확해보이긴해서 적어둡니다

---

좋습니다

---

근데 1마리와 가 맞지 않나요?